schemas/v1/openapi.yaml

```diff
@@ -287,6 +287,13 @@ components:
             The absence of this header implies that the server does not offer cascading lookups.
           schema:
             type: string
+        X-IPNI-Legacy-Cascade-Peering:
```

Comment:
:thought_balloon:   "Legacy Cascade"  sounds like a noun of some protocol, while it is just an implementation detail of IPNI doing fallback to Bitswap Peering.